lld/test/wasm/lto/Inputs/foo.ll

```diff
@@ -1,7 +1,17 @@
 target datalayout = "e-m:e-p:32:32-p10:8:8-p20:8:8-i64:64-n32:64-S128-ni:1:10:20"
 target triple = "wasm32-unknown-unknown"
 
-define void @foo() local_unnamed_addr {
+define void @foo() {
+entry:
+  ret void
+}
+
+define void @baz() {
+entry:
+  ret void
+}
+
+define void @quux() {
```

Comment:
Can you make a new input file rather than adding to this (since this one is just called `foo` I think its good it just defines `foo`)

Reply:
Oh sorry, `baz` is what I added... I thought it was already there. Will create another input file. Ignore the comment above

Reply:
I don't think you need to make a new stub file.  The stub file and the `.ll` file are not related explicitly.

Perhsonally I would just create a single new `.ll` file in under inputs to use for this test and not use the existings `Inputs/foo.ll`